locales-cs-CZ.xml

```diff
@@ -251,6 +251,7 @@
     </term>
 
     <!-- VERB ROLE FORMS -->
+    <term name="container-author" form="verb"/>
```

Comment:
The files are already doing this in several places (see e.g. `ar` above), so it should have come out already if they are being handled differently.